UnifiedMap/UnifiedMap.Droid/UnifiedMapRenderer.cs

```diff
@@ -372,7 +436,8 @@ public void SetSelectedAnnotation()
         {
             if (Map.SelectedItem is IMapPin selectedItem && _markers.TryGetValue(selectedItem, out var selectedMarker))
             {
-                OnMarkerClick(selectedMarker);
+                // Refacto -> OnMarkerClick is calling twice, cause of the Marker Click listener
+                //OnMarkerClick(selectedMarker);
```

Comment:
Are there any implications on commenting this line out?

If it can be commented out, can it be safely removed without causing a bug/regression?

Reply:
Id I comment it, it was just to remender where it was if someone find an issue.
But for me the event it's called twice no need to call it twice.